requirements.txt

```diff
@@ -0,0 +1,137 @@
+alabaster==0.7.12
```

Comment:
IMO this file should be called "requirements-release.txt" or something like that, to avoid users absentmindedly running `pip install -r requirements.txt` without realizing it could changing their environment's libraries. I think we should also change the name of docs/requirements.txt to docs/requirements-docker.txt (as it's only referenced in the Dockerfile above), to be more explicit

Reply:
Why do we need a specific docker requirements?

Reply:
We don't. We could get rid of that file and have the Dockerfile install from this file

Reply:
A downside of that approach is it will make the docker build process take a bit longer as each pinned library will need to be downloaded from PyPI instead of using the version that comes with the Docker base image (how much longer? maybe 1 or 2 minutes but not sure). To avoid that problem, we could store a corgie base image on dockerhub that uses the current pinned libraries, and then build from that base image instead.